src/helpers.test.js

```diff
@@ -82,4 +83,13 @@ describe('helpers', () => {
         expect(getFieldType(fooField)).toBe(String);
         expect(getFieldType(barField)).toBe(String);
     });
+
+    it('should return function name', () => {
+        const barFunction = function bar() {};
+        const fooFunction = function foo() {};
+        Object.defineProperty(fooFunction, 'name', { value: undefined });
+        expect(getFunctionName(barFunction)).toBe('bar');
+        expect(getFunctionName(String)).toBe('String');
```

Comment:
Please add expect for all default types:
- String
- Number
- Object
- Array
- Boolean
- Date

Reply:
I added it just right now

Reply:
Thanks :) Merged and published.